python/gen3authz/client/arborist/client.py

```diff
@@ -498,7 +498,7 @@ def delete_policy(self, path):
 
     def create_policy(self, policy_json, skip_if_exists=True):
         response = self.post(self._policy_url, json=policy_json)
-        if response.code == 409:
+        if response.code == 409 and skip_if_exists:
             # already exists; this is ok, but leave warning
             self.logger.warning(
                 "policy `{}` already exists in arborist".format(policy_json["id"])
```

Comment:
i would change this to
```
if response.code == 409:
 self.logger.warning()
 if skip_if_exists:
  return None
```
so that we still get the log if `skip_if_exists` is False

Reply:
I think the way it is right now, if code is 409 but `skip_if_exists` is False, then `response.successful` is False, so we (a) still get the log but also importantly (b) raise ArboristError? So it will get to the next block and then error out

Reply:
yes you get `could not create policy`
i meant that also seeing `policy `{}` already exists in arborist` can be useful
but i see now that `could not create policy` includes `response.error_msg`, which i assume would explain that the policy already exists, so i guess it's fine

Reply:
Ahhh. Yes: https://github.com/uc-cdis/arborist/blob/c58e30bfbf8b0297bc92d38a471469dfd3bce362/arborist/policy.go#L199